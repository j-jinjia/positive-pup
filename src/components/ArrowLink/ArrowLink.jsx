import "./ArrowLink.scss";
import arrow from "../../assets/svgs/arrow.svg";

const ArrowLink = ({ linkText, linkTo }) => {
  return (
    <a href={linkTo} className="arrow-link">
      {linkText}
      <img className="arrow-link__arrow" src={arrow} alt="" />
    </a>
  );
};

export default ArrowLink;
