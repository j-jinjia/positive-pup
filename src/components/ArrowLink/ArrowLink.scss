@use "../../styles/_variables" as *;

.link {
  color: $colour-primary;
  font-size: 1.25rem;
  display: flex;
  align-items: flex-end;
  gap: 1rem;
}
