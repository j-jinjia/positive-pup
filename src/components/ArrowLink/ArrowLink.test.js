import { customRender } from "../../utils/testUtils";
import ArrowLink from "./ArrowLink";

it("should match ArrowLink component snapshot", () => {
  const { container } = customRender(
    <ArrowLink linkText={"Test"} linkTo={"#Test"} />
  );

  expect(container).toMatchSnapshot();
});
