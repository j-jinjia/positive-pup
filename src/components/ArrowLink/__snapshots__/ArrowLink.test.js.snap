// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match ArrowLink component snapshot 1`] = `
<div>
  <a
    class="arrow-link"
  >
    <img
      alt=""
      class="arrow-link__arrow"
      src="arrow.svg"
    />
  </a>
</div>
`;
